examples/example-with-encryption/user-data-client.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This script is meant to be run in the User Data of each EC2 Instance while it's booting. The script uses the
+# run-consul script to configure and start Consul in client mode. Note that this script assumes it's running in an AMI
+# built from the Packer template in examples/consul-ami/consul.json.
+
+set -e
+
+# Send the log output from this script to user-data.log, syslog, and the console
+# From: https://alestic.com/2010/12/ec2-user-data-output/
+exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
+
+# These variables are passed in via Terraform template interplation
+/opt/consul/bin/run-consul --client --cluster-tag-key "${cluster_tag_key}" --cluster-tag-value "${cluster_tag_value}" ${gossip_encryption_configuration} ${rpc_encryption_configuration}
```

Comment:
Please add a comment that says something along the lines of "to keep this example simple, we are passing  the gossip encryption key in plain text to User Data, but in production usage, it should always be encrypted using a tool like Vault or KMS, and only decrypted, in memory, just before usage."